src/main/Config.cpp

```diff
@@ -1514,50 +1517,6 @@ Config::processConfig(std::shared_ptr<cpptoml::table> t)
             LIMIT_TX_QUEUE_SOURCE_ACCOUNT = true;
         }
 #endif
-        // PEER_FLOOD_READING_CAPACITY_BYTES (C): This is the initial credit
```

Comment:
Can we update these comments to reflect new changes if any? I think these comments were useful.

Reply:
nvm